### ISSUE TITLE
Cannot get cocoapods to install latest version

### ISSUE THREAD

**Issue by kallipigous** [devicekit/DeviceKit#173]
I've tried reinstalling, deleting everything I can think of but cocoapods insists on installing DeviceKit (1.12.0) rather than anything newer.

Any thoughts?

**Comment by Zandor300** [comment]
1.12.0 is the latest version currently.

**Comment by kallipigous** [comment]
Ah forgive me it says  pod 'DeviceKit', '~> 1.3' on the home page. 
I'll stop trying. I was hoping to get the new ipads included.

**Comment by Zandor300** [comment]
Oh, those changes haven't been released yet but if you really need it;
```ruby
pod 'DeviceKit', :git => 'https://github.com/devicekit/DeviceKit.git'
```
Might have to force an update somehow since the version number hasn't been incremented from 1.12.0.

**Comment by kallipigous** [comment]
Thanks. I'm happy to wait, I just wan't sure if my cocoapods was broken.

**Comment by denisenepraunig** [comment]
Hello @kallipigous - I've just published a new version 1.13.0 to CocoaPodsvwith all the latest changes from master. BR, Denise

**Comment by kallipigous** [comment]
Thank you
